settingshelper.py

```diff
@@ -139,10 +139,10 @@ def make_couchdb_tuples(self):
         return [self._make_couchdb_tuple(row) for row in self.couchdb_apps]
 
     def _make_couchdb_tuple(self, row):
-        if isinstance(row, six.string_types):
-            app_label, postfix = row, None
-        else:
+        if isinstance(row, tuple):
```

Comment:
Could make this a little more resilient to custom values in localsettings (`LOCAL_COUCHDB_APPS`) with `isinstance(row, (tuple, list))`

Reply:
Actually it looks like ```LOCAL_COUCHDB_APPS``` is never used, and never including in localsettings in ```commcare-cloud```.  I'm going to remove it.

Reply:
(Although I see your point about being resilient to ```list```s)